### PR TITLE
Failed to save `BlockOperation`s

### LINKED ISSUE #654: null operation value from `api/v1/accounts/{id}/operations`
Getting operations from `api/v1/accounts/{id}/operations` is not working.
It shows only null value.
I guess the reason may be that operation was not saved.

### PR DESCRIPTION
**Pull request by spikeekips**
### Github Issue

Resolves #654 

### Background

See #654 and when node is`BOOTING`, the `SavingBlockOperation` process hangs.

### Solution

There were problem in `SavingBlockOperation.CheckByBlock()`, if `blk.Transactions` is under 2, `numWorkers` was `0`.

### CODE REVIEW

File: lib/node/runner/block_operations.go [hunk 1]
```diff
@@ -110,41 +110,56 @@ func (sb *SavingBlockOperations) savingBlockOperationsWorker(id int, st *storage
 }
 
 func (sb *SavingBlockOperations) CheckByBlock(st *storage.LevelDBBackend, blk block.Block) (err error) {
+	if blk.Height > common.GenesisBlockHeight { // ProposerTransaction
+		if err = sb.CheckTransactionByBlock(st, blk, blk.ProposerTransaction); err != nil {
+			return
+		}
+	}
+	if len(blk.Transactions) < 1 {
+		return
```
Comment (Charleslee522) [inline lib/node/runner/block_operations.go]: 👍

**Comment by Geod24** [comment]
Can you add a test that would have failed before this patch ?

**Comment by soonkuk** [comment]
@spikeekips I checked working of api. I can get correct operation value.

**Comment by Geod24** [comment]
Will test soon with delayed ops

**Comment by spikeekips** [comment]
NOTE: I will wait until test result of @Geod24 or the other developers.

**Comment by Geod24** [comment]
Yes, it works 👍

**Pull request merged**